test/integration/scripts/test_stop.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2015-2025 Earth Sciences Department, BSC-CNS
+#
+# This file is part of Autosubmit.
+#
+# Autosubmit is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Autosubmit is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test for the autosubmit stop command"""
+
+from unittest.mock import patch
+
+import pytest
+from pytest_mock import MockerFixture
+
+from autosubmit.scripts.autosubmit import main
+
+_EXPID = "t111"
+
+
+def test_autosubmit_stop_command_invocation(autosubmit_exp, mocker: MockerFixture):
+    """
+    Test if the stop function was called by the CLI
+    """
+    autosubmit_exp(_EXPID, experiment_data={})
+
+    mocker.patch("sys.argv", ["autosubmit", "stop", "-y", _EXPID])
+
+    with patch("autosubmit.autosubmit.Autosubmit.stop") as mock_stop:
+        main()
+
+        mock_stop.assert_called_once()
+
+        passed_args = mock_stop.call_args[0]
+        assert passed_args[7] is True
```

Comment:
This is still brittle as if we add an arg before we will have to adjust this test.

From what I understood, this test only checks that the `-y` is parsed by argparse and results in the stop functon receiving the force yes. That's also covered by the test below, so I think we can remove this test entirely. Coverage showing it's not touching the changes in `stop()`

<img width="1407" height="1094" alt="image" src="https://github.com/user-attachments/assets/4d4eb23e-8487-4480-9a89-21f5cb0e0724" />

Or is it testing another part of the code not covered by `test_stop_bypass_prompt_confirmation`